.github/pyproject_version.py

```diff
@@ -0,0 +1,82 @@
+# Read package version in pyproject.toml and replace it in .conda/recipe.yaml
+
+import argparse
+import logging
+import re
+
+logging.basicConfig(level=logging.INFO, format='%(message)s')
+PACKAGE_VERSION = 'X.X.X'
+CORE_VERSION = '>=43,<44'
+NUMPY_VERSION = '>=1.24.3,<2'
+
+
+def get_versions():
+    '''
+    Read package version and deps in pyproject.toml
+    '''
+    openfisca_core_api = None
+    openfisca_france = None
+    with open('./pyproject.toml', 'r') as file:
+        content = file.read()
+    # Extract the version of openfisca_france
+    version_match = re.search(r'^version\s*=\s*"([\d.]*)"', content, re.MULTILINE)
+    if version_match:
+        openfisca_france = version_match.group(1)
+    else:
+        raise Exception('Package version not found in pyproject.toml')
+    # Extract dependencies
+    version = re.search(r'openfisca-core\[web-api\]\s*(>=\s*[\d\.]*,\s*<\d*)"', content, re.MULTILINE)
```

Comment:
```suggestion
    version = re.search(r'openfisca-core\s*(>=\s*[\d\.]*,\s*<\d*)"', content, re.MULTILINE)
```
Il faut changer cette expression régulière car l'import qui est fait ne mentionne pas `[web-api]` :
```
    'numpy >=1.24.2, <2.0',
    'openfisca-core >=43.0.0, <44.0.0',
```